ml4h/models.py

```diff
@@ -1145,31 +1563,62 @@ def _make_multimodal_multitask_model(
     return Model(inputs=list(inputs.values()), outputs=list(decoder_outputs.values()))
 
 
+def _transfer_layers_by_name(model_layers: str, freeze_model_layers: str, custom_dict: Dict[str, Any], m: Model):
```

Comment:
thanks for putting this in its own function!

Reply:
:)